src/option.cpp

```diff
@@ -229,12 +230,9 @@ static void unc_add_option(const char *name, uncrustify_options id, argtype_e ty
       value.max_val = 0;
       break;
 
-   case AT_TFI:
-      value.max_val = 2;
-      break;
-
    default:
-      fprintf(stderr, "FATAL: %s(%d): Illegal option type %d for '%s'\n", __func__, __LINE__, type, name);
+      fprintf(stderr, "FATAL: %s(%d): Illegal option type %d for '%s'\n",
+              __func__, __LINE__, static_cast<int>(type), name);
```

Comment:
Pedantically, I ought to be using `std::underlying_type` here, but #1852 is going to get rid of these...